### ISSUE TITLE
Use conda to install Decoil

### ISSUE THREAD

**Issue by madagiurgiu25** [madagiurgiu25/decoil-pre#4]
Create a conda installation for Decoil.

**Comment by zhober** [comment]
When can conda install be available? I really need it. Additionally, I have a new comment in [issue](https://github.com/madagiurgiu25/decoil-pre/issues/5#issuecomment-1903511163).

**Comment by madagiurgiu25** [comment]
Dear @zhober,

At the moment I do not have the capacities, but I have now included a installation from source (README.md), which requires `conda` and `pip`.

**Comment by zhober** [comment]
I'm trying to install Decoil from source in Python 3.8. However, I encountered an issue with the pickle5 package, which is only compatible with Python 3.5, 3.6, and 3.7, as mentioned in https://pypi.org/project/pickle5/0.0.11/.
Additionally, when attempting to install pybedtools, py2bit, and deeptoolsintervals using pip, I encountered an error message `× python setup.py bdist_wheel did not run successfully`. But these packages can be installed using conda.

**Comment by madagiurgiu25** [comment]
Ok, what OS system do you use?

**Comment by madagiurgiu25** [comment]
> I'm trying to install Decoil from source in Python 3.8. However, I encountered an issue with the pickle5 package, which is only compatible with Python 3.5, 3.6, and 3.7, as mentioned in https://pypi.org/project/pickle5/0.0.11/. Additionally, when attempting to install pybedtools, py2bit, and deeptoolsintervals using pip, I encountered an error message `× python setup.py bdist_wheel did not run successfully`. But these packages can be installed using conda.

You are right in the documentation is written that pickle5 is compatible with python < 3.8 but for some reason on my MacOS it got installed:

```
conda activate envdecoil
which python
# <mypath>/miniconda3/envs/envdecoil/bin/python

python -m pip freeze | grep pickle
# pickle5==0.0.11

python
#Python 3.8.0 | packaged by conda-forge | (default, Nov 22 2019, 19:11:19) 
#Type "help", "copyright", "credits" or "license" for more information.
>>> 

```

**Comment by zhober** [comment]
> Could you tell me which OS system did you use?

I install on Linux. It’s strange that you can install pickle5 on python3.8. I can’t find the source for python3.8 in https://pypi.org/project/pickle5/#files for windows Mac or Linux.

**Comment by madagiurgiu25** [comment]
I checked the documentation, for Python 3.8 pickle is in the standard library. I have now removed `pickle5` this from `requirements` and let me know if you can now install Decoil from source. Thank you.
